src/model/cloud-runner/cloud-runner.ts

```
@@ -64,7 +65,30 @@ class CloudRunner {
 
   private static setupSelectedBuildPlatform() {
     CloudRunnerLogger.log(`Cloud Runner platform selected ${CloudRunner.buildParameters.providerStrategy}`);
-    switch (CloudRunner.buildParameters.providerStrategy) {
+    // Detect LocalStack endpoints and reroute AWS provider to local-docker for CI tests that only need S3
```

ESLint

[error] Expected line before comment.

(lines-around-comment)